app/models/snp.rb

```diff
@@ -43,6 +46,15 @@ def self.update_frequencies
     end
   end
 
+  def self.update_phenotypes
+    max_age = 31.days.ago
+
+    Snp.select([:id, :phenotype_updated]).where(phenotype_updated < max_age)
+       .find_each do |snp|
+      Sidekiq::Client.enqueue(LinkSnpPhenotype, snp.id)
```

Comment:
Yes, they are. I'd prefer `perform_async` because I find it more readable, as it's less boilerplate code.